### PR TITLE
refactor: to follow style guidelines

### LINKED ISSUE #137: Refactor code to follow guidelines
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
Currently we have some templates which do not follow the guidelines specified [here](https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md) eg. templates/componenets/events/view/export/api-response.hbs which uses double quotes instead of single quotes inside helper.

**Expected behavior:**
All the templates must follow the style guide.

Please find all such files and refactor the code to follow the style-guide.

### PR DESCRIPTION
**Pull request by harshitagupta30**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This PR refactor the code to follow the [style](https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md) guidelines.

Fixes #137 
@niranjan94 @geekyd @CosmicCoder96 Please review!

### CODE REVIEW

File: app/templates/components/forms/register-form.hbs [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <form class="ui large form" autocomplete="off" {{action 'submit' on='submit'}}>
-  <div class="ui  aligned segment basic" style="margin: 0; padding: 0;">
```
Comment (niranjan94) [inline app/templates/components/forms/register-form.hbs]: Why ?

Comment (niranjan94) [inline app/templates/components/forms/register-form.hbs]: Please add space after `:` and add semicolon after each style

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 2]
```diff
@@ -17,9 +16,8 @@
       </div>
     </div>
   </div>
-
+  <br>
```
Comment (niranjan94) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: Why ?

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 3]
```diff
@@ -17,9 +16,8 @@
...
   {{#if sessionsSpeakers.enabled}}
-    <br>
```
Comment (niranjan94) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: Why ?

File: app/templates/components/nav-bar.hbs [hunk 4]
```diff
@@ -1,11 +1,11 @@
 <div class="mobile hidden row">
   <div class="ui menu">
     {{#link-to 'index' class='header item' activeClass=''}}
-      <h3 class="">{{config.appName}} </h3>
+      <h3 class="">{{config.appName}}</h3>
```
Comment (niranjan94) [inline app/templates/components/nav-bar.hbs]: Remove `class=""`

File: app/templates/components/public/call-for-speakers.hbs [hunk 5]
```diff
@@ -11,25 +11,22 @@
 </h1>
 <div class="ui divider"></div>
   <p>
-    The OpenTechSummit brings together the most exciting ideas and makers of the open-technology community. In 2017 the technology
-    pioneers in Potsdam will meet again on Ascension Day, the 25th of May, to present, discuss and network their projects and ideas in lectures and workshops.
+    {{t 'The OpenTechSummit brings together the most exciting ideas and makers of the open-technology community. In 2017 the technology
```
Comment (niranjan94) [inline app/templates/components/public/call-for-speakers.hbs]: Please don't put translation helpers for dummy data

File: app/templates/components/public/ticket-list.hbs [hunk 6]
```diff
@@ -21,7 +21,7 @@
             </div>
             <small class="ui gray-text tiny">{{ticket.description}}</small>
           </td>
-          <td>Thu, May 25</td>
+          <td>{{t 'Thu, May 25'}}</td>
```
Comment (niranjan94) [inline app/templates/components/public/ticket-list.hbs]: Please don't put translation helpers for dummy data

File: app/templates/components/widgets/forms/link-input.hbs [hunk 7]
```diff
@@ -16,17 +16,17 @@
       {{#if isChild}}
         <div class="ui icon buttons">
           {{#if canRemoveItem}}
-            <button type="button" class="ui sharp button red" {{action removeItem}}><i class="minus icon"></i></button>
+            <button type="button" class="ui sharp button red" {{action 'removeItem'}}><i class="minus icon"></i></button>
```
Comment (niranjan94) [inline app/templates/components/widgets/forms/link-input.hbs]: Why ? `removeItem` is a passed in action. Not a string.

File: app/templates/components/widgets/forms/link-input.hbs [hunk 8]
```diff
@@ -16,17 +16,17 @@
...
           {{/if}}
           {{#if canAddItem}}
-            <button type="button" class="ui button teal" {{action addItem}}><i class="plus icon"></i></button>
+            <button type="button" class="ui button teal" {{action 'addItem'}}><i class="plus icon"></i></button>
```
Comment (niranjan94) [inline app/templates/components/widgets/forms/link-input.hbs]: Revert this

File: app/templates/components/widgets/forms/ticket-input.hbs [hunk 9]
```diff
@@ -24,14 +24,14 @@
     <div class="field">
       <div class="ui icon buttons">
         <button type="button" class="ui button" {{action 'toggleSettings'}} data-content="Settings"><i class="setting icon"></i></button>
-        <button type="button" class="ui button" {{action removeTicket}} data-content="Delete"><i class="trash icon"></i></button>
+        <button type="button" class="ui button" {{action 'removeTicket'}} data-content="Delete"><i class="trash icon"></i></button>
```
Comment (niranjan94) [inline app/templates/components/widgets/forms/ticket-input.hbs]: Revert this. These are all actions passed in. Please don't make such changes blindly without checking the code.

File: app/templates/components/widgets/forms/ticket-input.hbs [hunk 10]
```diff
@@ -24,14 +24,14 @@
...
       </div>
       <div class="ui icon buttons right floated">
         {{#if canMoveUp}}
-          <button type="button" class="ui button" {{action moveTicketUp}} data-content="Move ticket up"><i class="up arrow icon"></i></button>
+          <button type="button" class="ui button" {{action 'moveTicketUp'}} data-content="Move ticket up"><i class="up arrow icon"></i></button>
```
Comment (niranjan94) [inline app/templates/components/widgets/forms/ticket-input.hbs]: Revert this. These are all actions passed in. Please don't make such changes blindly without checking the code.

File: app/templates/components/widgets/forms/ticket-input.hbs [hunk 11]
```diff
@@ -24,14 +24,14 @@
...
         {{/if}}
         {{#if canMoveDown}}
-          <button type="button" class="ui button" {{action moveTicketDown}} data-content="Move ticket down"><i class="down arrow icon"></i></button>
+          <button type="button" class="ui button" {{action 'moveTicketDown'}} data-content="Move ticket down"><i class="down arrow icon"></i></button>
```
Comment (niranjan94) [inline app/templates/components/widgets/forms/ticket-input.hbs]: Revert this. These are all actions passed in. Please don't make such changes blindly without checking the code.

File: app/templates/events/live.hbs [hunk 12]
```diff
@@ -1 +1 @@
-{{events/events-table columns=columns data=data style='margin-top: 20px;'}}
+{{events/events-table columns=columns data=data style='margin-top:20px'}}
```
Comment (niranjan94) [inline app/templates/events/live.hbs]: Remove `style='margin-top:20px'`

File: app/templates/events/view.hbs [hunk 13]
```diff
@@ -2,7 +2,7 @@
   <div class="ui grid stackable">
     <div class="row">
       <div class="four wide column">
-        <h2 class="ui header">Fossasia Test</h2>
+        <h2 class="ui header">{{t 'Fossasia Test'}}</h2>
```
Comment (niranjan94) [inline app/templates/events/view.hbs]: Please don't put translation helpers for dummy data. Doesn't make sense.

File: app/templates/events/view/sessions.hbs [hunk 14]
```diff
@@ -40,30 +40,30 @@
             </div>
           </td>
           <td>
-            Test Session
+            {{t 'Test Session'}}
```
Comment (niranjan94) [inline app/templates/events/view/sessions.hbs]: Please don't put translation helpers for dummy data. Doesn't make sense.

File: app/templates/events/view/tickets/orders.hbs [hunk 15]
```diff
@@ -113,7 +113,7 @@
               </div>
             </td>
             <td>
-              US $ {{order.amount}}
+              {{t 'US'}}${{order.amount}}
```
Comment (niranjan94) [inline app/templates/events/view/tickets/orders.hbs]: Don't put translation helpers for all strings blindly... We'll not be translating currency codes.

File: app/templates/my-tickets/index.hbs [hunk 16]
```diff
@@ -2,9 +2,9 @@
   <div class="sixteen wide column">
     {{#each model as |event|}}
       {{#event-card event=event isWide=true shareEvent=(action 'shareEvent')}}
-        <span>Free Ticket</span>
+        <span>{{t 'Free Ticket'}}</span>
```
Comment (niranjan94) [inline app/templates/my-tickets/index.hbs]: Please don't put translation helpers for dummy data. Doesn't make sense.

**Comment by harshitagupta30** [comment]
@niranjan94 I have made all the required changes. Please review it again. Thanks.

File: app/templates/components/errors/not-found.hbs [hunk 17]
```diff
@@ -1,9 +1,9 @@
 <br>
-<h1 class="text muted weight-300" style="margin-bottom: 0;">404 {{t 'Not Found'}}</h1>
-<h1 style="font-size: 3.5em; " class="weight-300">{{t "Oops, the page you're"}}<br> {{t 'looking for does not'}}<br> {{t 'exist'}}.</h1>
+<h1 class="text muted weight-300" style="margin-bottom:0">404 {{t 'Not Found'}}</h1>
```
Comment (geekyd) [inline app/templates/components/errors/not-found.hbs]: Please add space after `:` & `;` in the end.

File: app/templates/components/errors/server-error.hbs [hunk 18]
```diff
@@ -1,9 +1,13 @@
 <br>
-<h1 class="text muted weight-300" style="margin-bottom: 0;">500 {{t 'Internal Server Error'}}</h1>
-<h1 style="font-size: 3.5em; " class="weight-300">{{t "Oops, the page you're"}}<br> {{t 'looking for caused an error'}}<br>{{t 'on our servers'}}.</h1>
+<h1 class="text muted weight-300" style="margin-bottom:0">500 {{t 'Internal Server Error'}}</h1>
```
Comment (geekyd) [inline app/templates/components/errors/server-error.hbs]: Please add space after `:` & `;` in the end.

File: app/templates/components/events/view/export/download-zip.hbs [hunk 19]
```diff
@@ -10,16 +10,16 @@
...
   </div>
   <div class="field">
-    {{ui-checkbox class="toggle" label=(t 'Audio')}}
+    {{ui-checkbox class='toggle' label=(t 'Audio')}}
   </div>
   <div class="field">
-    {{ui-checkbox class="toggle" label=(t 'Document')}}
+    {{ui-checkbox class='toggle' label=(t 'Document')}}
   </div>
   <div class="ui basic segment" style="padding-left:0">
```
Comment (geekyd) [inline app/templates/components/events/view/export/download-zip.hbs]: Please add space after `:` & `;` in the end.

File: app/templates/components/widgets/forms/location-input.hbs [hunk 20]
```diff
@@ -44,7 +44,7 @@
     </div>
     <div class="column" style="">
```
Comment (geekyd) [inline app/templates/components/widgets/forms/location-input.hbs]: Please remove `style=""`

**Comment by harshitagupta30** [comment]
@niranjan94 @geekyd please review it again! Thanks.

File: app/templates/components/ui-table/global-filter.hbs [hunk 21]
```diff
@@ -1,5 +1,5 @@
 <div class="ui column models-table-global-filter">
   <div class="ui input ">
-    {{input type="text" value=filterString class="" enter="" placeholder=messages.searchPlaceholder}}
+    {{input type='text' value=filterString class='' enter='' placeholder=messages.searchPlaceholder}}
```
Comment (geekyd) [inline app/templates/components/ui-table/global-filter.hbs]: Please remove `class`

File: app/templates/events/live.hbs [hunk 22]
```diff
@@ -1 +1 @@
-{{events/events-table columns=columns data=data style='margin-top: 20px;'}}
```
Comment (geekyd) [inline app/templates/events/live.hbs]: Why?

Comment (harshitagupta30) [inline app/templates/events/live.hbs]: @geekyd Hi Thanks! @niranjan94 had suggested to remove this. you can check outdated comments

**Comment by harshitagupta30** [comment]
@geekyd @niranjan94 @CosmicCoder96 I have made all the requested changes. Please review!

File: app/templates/components/explore/side-bar.hbs [hunk 23]
```diff
@@ -61,8 +61,7 @@
     </span>
     <div class="content menu">
       {{#each dateRanges as |dateRange|}}
-        <a href="#" class="link item {{if (eq filters.dateRange dateRange.key) 'active'}}" {{action 'selectDateRange'
-                                                                                                    dateRange.key}}>
+        <a href="#" class="link item {{if (eq filters.dateRange dateRange.key) 'active'}}" {{action 'selectDateRange' dateRange.key}}>
```
Comment (geekyd) [inline app/templates/components/explore/side-bar.hbs]: Please make it  `{{action selectDateRange dateRange.key}}`

**Comment by harshitagupta30** [comment]
@geekyd updated! please review.

**Comment by CosmicCoder96** [comment]
@harshitagupta30 PLease resolve conflicts so I can merge.

**Comment by harshitagupta30** [comment]
@CosmicCoder96 Resolved! :+1:

File: app/templates/components/ui-table/header-row-filtering.hbs [hunk 24]
```diff
@@ -11,9 +11,9 @@
             {{#if column.useFilter}}
               <div class="ui input {{if column.filterUsed 'has-feedback'}}">
                 {{#if column.filterWithSelect}}
-                  {{models-select options=column.filterOptions cssPropertyName=column.cssPropertyName value=column.filterString class=(concat classes.input " changeFilterForColumn")}}
+                  {{models-select options=column.filterOptions cssPropertyName=column.cssPropertyName value=column.filterString class=(concat classes.input 'changeFilterForColumn')}}
                 {{else}}
-                  {{input type="text" value=column.filterString class=classes.input enter="" placeholder=""}}
+                  {{input type='text' value=column.filterString class=classes.input enter='' placeholder=''}}
```
Comment (niranjan94) [inline app/templates/components/ui-table/header-row-filtering.hbs]: remove `enter='' placeholder=''`

File: app/templates/components/ui-table/global-filter.hbs [hunk 25]
```diff
@@ -1,5 +1,5 @@
 <div class="ui column models-table-global-filter">
   <div class="ui input ">
-    {{input type="text" value=filterString class="" enter="" placeholder=messages.searchPlaceholder}}
+    {{input type='text' value=filterString enter='' placeholder=messages.searchPlaceholder}}
```
Comment (niranjan94) [inline app/templates/components/ui-table/global-filter.hbs]: remove `enter=''`

File: app/templates/events/live.hbs [hunk 26]
```diff
@@ -1 +1,2 @@
-{{events/events-table columns=columns data=model style='margin-top: 20px;'}}
+{{events/events-table columns=columns data=model}}
+{{events/events-table columns=columns data=data}}
```
Comment (niranjan94) [inline app/templates/events/live.hbs]: Remove this

**Comment by harshitagupta30** [comment]
@niranjan94 Please review it again! thanks

**Pull request merged**